packages/internet-header/src/components/post-internet-header/logo-animation/logo-animation.ts

```diff
@@ -0,0 +1,54 @@
+import { debounce } from 'throttle-debounce';
+
+export const registerLogoAnimationObserver = (
+  target: HTMLElement,
+  headerRef: HTMLSwisspostInternetHeaderElement,
+) => {
+  /**
+   * Set intersection ratio as CSS custom property
+   */
+  const handleScroll = () => {
+    let scale = 1;
+    // Minus 1px border at the bottom that the logo is not covering
+    const adjustedHeaderHeight = headerRef.clientHeight - 1;
+
+    // If meta navigation is not visible (mobile, not configured), scale should just be 1
+    if (target.clientHeight > 0) {
+      scale = Math.max(
+        (adjustedHeaderHeight - Math.max(window.scrollY, 0)) /
+          (adjustedHeaderHeight - target.clientHeight),
+        1,
+      );
+    }
+    headerRef.style.setProperty('--logo-scale', scale.toString());
+  };
+
+  const debounced = debounce(150, handleScroll);
```

Comment:
Would it make the transition smoother, if you reduce/remove the debounce?

Reply:
For resizing the window, animation smootheness is not important.